dex/testing/loadbot/sidestacker.go

```diff
@@ -34,36 +35,43 @@ type sideStacker struct {
 	// each epoch. If there are already enough standing limit orders,
 	// sideStacker will place orders to match immediately.
 	ordsPerEpoch int
+	// oscilator will cause the side stacker rate to move up and down over
+	// time. Used for a sideways market. Shared between side stackers so
+	// that they push in the same direction.
+	oscillator *uint64 // atomic
+	// active designates a side stacker that can write to the oscillator.
+	active bool
 }
 
 var _ Trader = (*sideStacker)(nil)
 
-func newSideStacker(seller bool, numStanding, ordsPerEpoch int, node string, metered bool, log dex.Logger) *sideStacker {
+func newSideStacker(numStanding, ordsPerEpoch int, node string, seller, metered, active bool, oscillator *uint64, log dex.Logger) *sideStacker {
 	return &sideStacker{
 		log:          log,
 		node:         node,
 		seller:       seller,
 		metered:      metered,
 		numStanding:  numStanding,
 		ordsPerEpoch: ordsPerEpoch,
+		active:       active,
+		oscillator:   oscillator,
 	}
 }
 
 func runSideStacker(numStanding, ordsPerEpoch int) {
 	if numStanding < ordsPerEpoch {
 		panic("numStanding must be >= minPerEpoch")
 	}
-	go blockEvery2()
-
+	var oscillator uint64
 	var wg sync.WaitGroup
 	wg.Add(2)
 	go func() {
 		defer wg.Done()
-		runTrader(newSideStacker(true, numStanding, ordsPerEpoch, alpha, false, log.SubLogger("STACKER:0")), "STACKER:0")
+		runTrader(newSideStacker(numStanding, ordsPerEpoch, alpha, true, false, true, &oscillator, log.SubLogger("STACKER:0")), "STACKER:0")
```

Comment:
> Is it intent that only the seller is an active oscillator, not the buyer?

No, only that one is shared across all of the sidestackers so that they are pushing in the same direction. Is there a more obvious mechanism to do this with?

Reply:
Using the named variables and wrapping after.

Reply:
> No, only that one is shared across all of the sidestackers so that they are pushing in the same direction. Is there a more obvious mechanism to do this with?

I was referring to `active = false` for the buyer vs. `active = true` for the seller.  If it's not an active oscillator, that means... it doesn't oscillate?

Reply:
Maybe active is the wrong variable name to use. The active sidetracker reads and writes to the oscillator while the others only read from it. Doing it this way so only one is changing the values of it, so it can randomly go a different direction (increasing rate vs decreasing rate) and the others follow.

maybe just `oscillatorWrite`?

Reply:
Changing to `oscillatorWrite` and hope I understood the confusion and this fixes it.

The TS `active` does indicate that the non-active are just drones and the active one tells them what to do. Designed that way to make huge orders possible while ignoring amount limitations.